src/app/apps/lti/views.py

```diff
@@ -70,6 +70,13 @@ def _do_on_success(self, lti_request: LTI, *args, **kwargs) -> HttpResponse:
             logger.debug("LTI user is not valid: %s", lti_user_form.errors)
             raise PermissionDenied
 
+        if (
+            lti_request.get_param("lti_message_type")
+            != LTIMessageType.LAUNCH_REQUEST.value
+        ):
+            logger.debug("LTI message type is not valid.")
```

Comment:
We don't usually add dot at the end of debug / log messages.

```suggestion
            logger.debug("LTI message type is not valid")
```

Reply:
If you agree, this applies everywhere.

Reply:
Alright, fixed everywhere.